config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -5013,6 +5013,14 @@ spec:
                     - enabled
                     type: object
                 type: object
+              logMonitoring:
```

Comment:
you have to run `make doc/api-ref` to update docs.

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3595/commits/d28e3b77f6b09a62c54494ddc435a1e7538c5ee5